manuscript/ha-docker-swarm/shared-storage-ceph.md

```diff
@@ -155,21 +155,24 @@ Grab the secret associated with the new user (you'll need this for the /etc/fsta
 ceph-authtool /etc/ceph/keyring.dockerswarm -p -n client.dockerswarm
 ```
 
+!!! note
+    These commands need to be executed from within the running `ceph-mon` container. To do this, we will use the `docker exec` command to open a shell *inside* the running container in order to run the commands above:
+    ```docker exec -it ceph-mon /bin/sh``` 
+
 ### Mount MDS volume
 
 On each node, create a mountpoint for the data, by running ```mkdir /var/data```, add an entry to fstab to ensure the volume is auto-mounted on boot, and ensure the volume is actually _mounted_ if there's a network / boot delay getting access to the gluster volume:
 
 ```
 mkdir /var/data
 
-MYHOST=`hostname -s`
-echo -e "
+echo " \
 # Mount cephfs volume \n
-$MYHOST:6789:/      /var/data/      ceph      \
+<node_IP>:6789:/      /var/data/      ceph      \
 name=dockerswarm\
 ,secret=<YOUR SECRET HERE>\
 ,noatime,_netdev,context=system_u:object_r:svirt_sandbox_file_t:s0 \
-0 2" >> /etc/fstab
+0 2" | sudo tee -a /etc/fstab
```

Comment:
Updated to use `tee` to append to fstab. Previous command replaced the existing contents of fstab.

Reply:
``>`` should replace the contents of files not ``>>``.

Reply:
Not sure what to say here. The command in the recipe overwrote my existing fstab file (I know this because I had previously set up an fstab entry to connect to an NFS share on one of my other machines prior to completing the recipe and it was no longer there).

Reply:
That ``>>`` works the same way as tee.

```
$ echo 1 > test
$ echo 2 >> test
$ cat test
1
2
$ echo 2 > test
$ cat test
2
```

Reply:
I concur that `>>` should always append. As a guideline, I'd prefer the cookbook to provide the _simplest_ way for a user to achieve an outcome. Unless we can demonstrate that `>>` is overwriting files, then I think we should stick with it, as the simpler approach

Reply:
Copy that...reverted. Maybe I mis-typed something on the original trip through.